### PR TITLE
Add tests for rnaseq

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/references/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/references _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: tests/rnaseq.nf.test [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    script "../main.nf"
+    tag "pipeline"
+
+    test("Run with profile test for rnaseq | --input assets/genomes/test/pipelines/default.yml") {
+
+        when {
+            params {
+                input  = "${projectDir}/assets/genomes/test/pipelines/default.yml"
+                outdir = "$outputDir"
+                tools  = 'bowtie,bowtie2,faidx,hisat2,hisat2_extractsplicesites,kallisto,make_transcripts_fasta,rsem_preparereference_genome,salmon,star'
```
Comment (edmundmiller) [inline tests/rnaseq.nf.test:13]: Should we break these out into their own params? I feel like it'll be easy to miss one with this long of a string.

Comment (edmundmiller) [reply tests/rnaseq.nf.test:13]: I can't remember if you can check these at the schema level like this? I think that's what happen in Demultiplex right?

**Review by edmundmiller — APPROVED** [review]
I love watching a pipeline get built in this post nf-test, and all the other tooling things era 😍

File: tests/rnaseq.nf.test (hunk 1 above)
Comment (maxulysse) [reply tests/rnaseq.nf.test:13]: We can decide later at some point, but I'd rather have a single params for these than multiple

**Pull request merged**